packages/sources/iex-cloud-test/src/endpoint/crypto.ts

```diff
@@ -0,0 +1,97 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import {
+  CryptoPriceEndpoint,
+  PriceEndpointParams,
+} from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { PriceEndpointInputParameters } from '@chainlink/external-adapter-framework/adapter'
+
+interface ResponseSchema {
+  symbol: string
+  primaryExchange: string
+  sector: string
+  calculationPrice: string
+  high: string
+  low: string
+  latestPrice: string
+  latestSource: string
+  latestUpdate: number
+  latestVolume: string
+  previousClose: string
+}
+
+export type CryptoEndpointTypes = {
+  Request: {
+    Params: PriceEndpointParams
+  }
+  Response: SingleNumberResultResponse
+  Settings: typeof config.settings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'asset', 'symbol'],
+    description: 'The symbol of symbols of the currency to query',
+    required: true,
+    type: 'string',
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    description: 'The symbol of the currency to convert to',
+    required: true,
+    type: 'string',
+  },
+} satisfies InputParameters & PriceEndpointInputParameters
+
+export const httpTransport = new HttpTransport<CryptoEndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params,
```

Comment:
Since we aren't batching on this endpoint, we should set params to the individual param here.
```suggestion
        params: [param],
```